### PR TITLE
Additional mount unittest

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

### CODE REVIEW

**Comment by gliptak** [comment]
@jsafrane @ddebroy please /ok-to-test

File: pkg/driver/mount_test.go [hunk 1]
```diff
@@ -41,6 +41,10 @@ func TestMakeDir(t *testing.T) {
 		t.Fatalf("Expect no error but got: %v", err)
 	}
 
+	if mountObj.MakeDir(targetPath) != nil {
```
Comment (leakingtapan) [inline pkg/driver/mount_test.go:44]: Duplicated code?

File: pkg/driver/mount_test.go [hunk 2]
```diff
@@ -64,8 +68,52 @@ func TestMakeFile(t *testing.T) {
 		t.Fatalf("Expect no error but got: %v", err)
 	}
 
+	if mountObj.MakeFile(targetPath) != nil {
```
Comment (leakingtapan) [inline pkg/driver/mount_test.go:71]: Same above

File: pkg/driver/mount_test.go [hunk 3]
```diff
@@ -64,8 +68,52 @@ func TestMakeFile(t *testing.T) {
...
+	}
+	defer os.RemoveAll(dir)
+
+	targetPath := filepath.Join(dir, "notafile")
+
+	var (
+		mountObj = newNodeMounter()
+	)
+
+	if exists, _ := mountObj.ExistsPath(targetPath); exists {
```
Comment (leakingtapan) [inline pkg/driver/mount_test.go:95]: Add a check for error

File: pkg/driver/mount_test.go [hunk 4]
```diff
@@ -64,8 +68,52 @@ func TestMakeFile(t *testing.T) {
...
+		t.Fatalf("Expected true")
```
Comment (leakingtapan) [inline pkg/driver/mount_test.go:96]: NIT: 
```suggestion
		t.Fatalf("Expected file %s to not exist", targetPath)
```

File: pkg/driver/mount_test.go [hunk 5]
```diff
@@ -64,8 +68,52 @@ func TestMakeFile(t *testing.T) {
...
+		t.Fatalf("Expect no error but got: %v", err)
+	}
+
 	if exists, err := mountObj.ExistsPath(targetPath); !exists {
 		t.Fatalf("Expect no error but got: %v", err)
 	}
 
 }
+
+func TestExistsPath(t *testing.T) {
```
Comment (leakingtapan) [inline pkg/driver/mount_test.go:81]: Could you add a happy case for this?

File: pkg/driver/mount_test.go (hunk 1 above)
Comment (gliptak) [reply pkg/driver/mount_test.go:44]: No, intentional. Different code path when exists

File: pkg/driver/mount_test.go (hunk 2 above)
Comment (gliptak) [reply pkg/driver/mount_test.go:71]: No, intentional. Different code path when exists

File: pkg/driver/mount_test.go (hunk 5 above)
Comment (gliptak) [reply pkg/driver/mount_test.go:81]: The happy case is covered in the above two tests. Do you see a value for a separate one?

File: pkg/driver/mount_test.go (hunk 3 above)
Comment (gliptak) [reply pkg/driver/mount_test.go:95]: updated

**Comment by gliptak** [comment]
@leakingtapan other updates you would like to see?

**Pull request merged**